setup.py

```diff
@@ -12,12 +12,9 @@
     author="Nathaniel J. Smith",
     author_email="njs@pobox.com",
     license="MIT",
-    packages=find_packages(),
+    packages=find_packages(exclude=["h11.tests"]),
     package_data={'h11': ['py.typed']},
     url="https://github.com/python-hyper/h11",
-    # This means, just install *everything* you see under h11/, even if it
-    # doesn't look like a source file, so long as it appears in MANIFEST.in:
-    include_package_data=True,
```

Comment:
Did you verify the sdist tarball has the tests? If not, please do as others rely on their presence there

Reply:
🤦missed that. Saw the comment about reducing artifact size and was pretty sure this boolean disabled MANIFEST.in but maybe tooling has improved to not conflate the two.